engine/common/identification.c

```diff
@@ -284,7 +284,7 @@ void ID_TestCPUInfo_f( void )
 	bloomfilter_t value = 0;
 
 	if( ID_ProcessCPUInfo( &value ) )
-		Msg( "Got %016llX\n", value );
+		Msg( "Got %016lX\n", value );
```

Comment:
bloomfilter_t is long long typedef.

Reply:
Are you sure?

GCC 9.2 prints a warning:

../engine/common/identification.c:287:19: warning: format ‘%llX’ expects argument of type ‘long long unsigned int’, but argument 2 has type ‘bloomfilter_t’ {aka ‘long unsigned int’} [-Wformat=]

Here is a screenshot:
![Снимок экрана_2019-10-22_19-39-16](https://user-images.githubusercontent.com/22560266/67303595-b57bb080-f503-11e9-8d20-c681d616369c.png)

Reply:
https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/identification.c#L32
https://github.com/FWGS/xash3d-fwgs/blob/master/common/xash3d_types.h#L35

Reply:
@iZarif

Are you compiling 64-bit engine?bMaybe this is a compiler bug then. Because
long long and long int are 64-bit on LP64(amd64 on Linux) and long long is 64 and long int 32 on ILP32(i386) and LLP64(amd64 on Windows).

I think we need to use stdint ASAP...

Reply:
Yes I compile a 64 bit engine

Reply:
@a1batross probably, long and long long defined as different types in gcc headers for 64bit.
@iZarif https://www.viva64.com/ru/t/0012/

Reply:
@nekonomicon they can't be _defined_ in _headers_, as they are builtin types. 

That's why I'm thinking it's a compiler bug.

Reply:
@nekonomicon
Do you think this link is directly related to this PR?

Reply:
@iZarif Why not?
Your PR breaks valid code.

Reply:
@nekonomicon 
Could you then give a link to the English text?